backend/api/v1/auth_service.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/pkg/errors"
 
+	"github.com/dlclark/regexp2"
```

Comment:
Why do we need regexp2 instead of using regexp?

Reply:
Cause the original regexp not support the syntax

```
error parsing regexp: invalid or unsupported Perl syntax: `(?=`
```